doc/lua_api.txt

```diff
@@ -3437,6 +3495,16 @@ angles in radians.
     * If `up` is omitted, the roll of the returned vector defaults to zero.
     * Otherwise `direction` and `up` need to be vectors in a 90 degree angle to each other.
 
+Further helpers
+---------------
+
+There are more helper functions involving vectors, but they are listed elsewhere
+because they only work on specific sorts of vectors or involve things that are not
+vectors.
+For example:
```

Comment:
Again with the ragged alignment, and as rubenwardy said earlier, put two newlines before all markdown lists.

Reply:
- Same as above.
- List fixed.